### PR TITLE
fix(components,releases,projects,licenses): fix NullPointerException in edit pages

### PR DESCRIPTION
**Pull request by alexbrdn**
- Using enum values directly in EL expressions doesn't work and that caused the NPE. Converting the Enum to a JSP variable beforehand allows working around the problem.
- removed unnecessary JS dictionaries with permissions as all objects already get filled with current user's permissions before edit.

closes sw360/sw360portal#553

### CODE REVIEW

File: frontend/sw360-portlet/src/main/webapp/html/projects/edit.jsp [hunk 1]
```diff
@@ -156,7 +155,7 @@
                           '<core_rt:if test="${not empty project.attachments}" ><li><sw360:out value="${project.attachmentsSize}"/> attachments</li></core_rt:if>'  +
                           '<core_rt:if test="${not empty project.linkedProjects or not empty project.releaseIdToUsage or not empty project.attachments}" ></ul></core_rt:if>' +
                           '</div>'+
-                          '<div ' + styleAsHiddenIfNeccessary(permissions.delete) + '><hr><label class=\'textlabel stackedLabel\'>Comment your changes</label><textarea id=\'moderationDeleteCommentField\' class=\'moderationCreationComment\' placeholder=\'Comment on request...\'></textarea></div>';
+                          '<div ' + styleAsHiddenIfNeccessary(${component.permissions[DELETE] == true}) + '><hr><label class=\'textlabel stackedLabel\'>Comment your changes</label><textarea id=\'moderationDeleteCommentField\' class=\'moderationCreationComment\' placeholder=\'Comment on request...\'></textarea></div>';
```
Comment (ChristophNiehoff) [inline frontend/sw360-portlet/src/main/webapp/html/projects/edit.jsp]: Shouldn't this be `${project.permissions[DELETE] == true}` at the project edit page?  I think there is a copy/paste-error in all the `styleAsHiddenIfNeccessary`-calls.
Do you really need the `== true` at this point?

**Comment by ChristophNiehoff** [comment]
With this change we can also remove again the addition of `currentUser` to the request in all the edit pages.

**Comment by alexbrdn** [comment]
Removed the `currentUser`

**Comment by ChristophNiehoff** [comment]
@alexbrdn 
Maybe we have been to fast in approving this pull request, meaning I was too fast with the code review. During testing I realized that  the evaluation of writing permissions for licenses is not correct, such that also admins will be asked for a moderation request comment.  I think the reason for this lies in the fact that `${licenseDetail.permissions[WRITE]}` is used to evaluate the permissions. But in the `LicensePortlet.prepareDetailView` this `License`-object is not filled with a permissions-map. `moderationLicenseDetail` should rather be used to get the permissions for licenses.

**Comment by alexbrdn** [comment]
Oh, well... I feel a little embarrassed for merging it too quickly. As this does not appear to be a showstopper, I'm going to make myself a cup of tea and fix it slowly and thoroughly.

**Pull request merged**